### ISSUE TITLE
DataGrid: property to expose count of unfiltered/filtered records  

### ISSUE THREAD

**Issue by mitchabaza** [stsrki/Blazorise#784]
It would be nice to get access to the total record count of the data set underlying the DataGrid as well as the count of the records currently being displayed if there any active filters.

**Comment by stsrki** [comment]
Hi @mitchabaza 

If we're talking about in-memory data than you already have that data at the time you provide it to datagrid. It will also be changed every time data is changed. So it's simple as this: `var count = youLData.Count();`

If we're talking about manual `ReadData` then you also need to provide that information to the datagrid by setting the TotalItems property.  https://blazorise.com/docs/extensions/datagrid/#large-data-example

The only missing information is number of currently visible items.

**Comment by mitchabaza** [comment]
@stsrki You are correct that total record count can be obtained from the underlying collection. Would you like me to amend the feature request?

**Comment by KapteinBabbalas** [comment]
Would be great to get the Filtered data item count especially if you are using the Filter Templates in combination with custom filter.
Is there any way to read the Item count or even the actual items? A scenario I have is where the user filters the list and then wants to perform a bulk operation on that subset.
